citi-onboarding/acervusej-onepage
# Feat/services

Things I've done:

-  Create the Our Services section responsively
-  Get the information for each Our Services item registered in the database, via the back-end route
-  Use the information that was picked up from the back-end on the Our Services carousel

How to test it:

-  First download the node modules on the client and the server. If you have already done that, then:
-  Change the values on .env.
-  Go to client on the terminal (cd client) and put npm run start.
-   Go to server on the terminal (cd server) and put node .\index.js
-  On the browser write http://localhost:3000/.
-  See if the "ourServices" works in mobile and desktop view.
-  Look at the code to see if it has everything that it should have.
